aclk/aclk_rx_msgs.c

```diff
@@ -496,6 +498,15 @@ void aclk_handle_new_cloud_msg(const char *message_type, const char *msg, size_t
         }
         return;
     }
+
+#ifdef NETDATA_INTERNAL_CHECKS
+    char *json = protomsg_to_json(msg, msg_len, msg_descriptor->name);
+    if (json) {
```

Comment:
you do not need to test this, because inside `protomsg_to_json` when `strdupz` cannot allocate the memory, it will call `fatal`.

Reply:
thanks, yes you are right before that function returned NULL in error case but later I changed it to always return string (error description